app/viewmodels/VatRegistrationDetailsSummary.scala

```diff
@@ -0,0 +1,101 @@
+/*
+ * Copyright 2023 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package viewmodels
+
+import formats.Format.dateOfRegistrationFormatter
+import models.UserAnswers
+import play.api.i18n.Messages
+import play.twirl.api.HtmlFormat
+import uk.gov.hmrc.govukfrontend.views.viewmodels.content.HtmlContent
+import uk.gov.hmrc.govukfrontend.views.viewmodels.summarylist.SummaryListRow
+import viewmodels.govuk.summarylist._
+import viewmodels.implicits._
+
+object VatRegistrationDetailsSummary {
```

Comment:
Maybe 

```scala
  def rowBusinessName(answers: UserAnswers)(implicit messages: Messages): Option[SummaryListRow] =
    answers.vatInfo.flatMap {
      answer =>
        answer.organisationName.map { organisationName =>
//          val value = HtmlFormat.escape(organisationName).toString
//
//          SummaryListRowViewModel(
//            key = "vatRegistrationDetails.checkYourAnswers.organisationName",
//            value = ValueViewModel(HtmlContent(value))
//          )

          createNameSummaryListRow(organisationName, "organisationName")
        }
    }

  private def createNameSummaryListRow(name: String, checkYouAnswersKey: String)(implicit messages: Messages): SummaryListRow = {
    val value = HtmlFormat.escape(name).toString

    SummaryListRowViewModel(
      key = s"vatRegistrationDetails.checkYourAnswers.$checkYouAnswersKey",
      value = ValueViewModel(HtmlContent(value))
    )
  }


  def rowIndividualName(answers: UserAnswers)(implicit messages: Messages): Option[SummaryListRow] =
    answers.vatInfo.flatMap {
      answer =>

        answer.individualName.map { individualName =>
//          val value = HtmlFormat.escape(individualName).toString
//
//          SummaryListRowViewModel(
//            key = "vatRegistrationDetails.checkYourAnswers.individualName",
//            value = ValueViewModel(HtmlContent(value))
//          )


          createNameSummaryListRow(individualName, "individualName")
        }
    }
```

Usually I would follow the rule of 3 but my duplication detection was firing with the game of spot the difference.

Reply:
Done